src/transduce.jl

```diff
@@ -149,25 +149,25 @@ end
     end
 
     # combineblock
-    offsetb = (igl - 1) * groupsize()[1]
-    bound = max(0, nbasecases - offsetb)
+    offsetb = @uniform (igl - 1) * groupsize()[1]
+    bound = @uniform max(0, nbasecases - offsetb)
 
     # shared mem for a complete reduction
     shared = @localmem(T, (2 * groupsize()[1]))
     @inbounds shared[ill] = acc
 
-    m = ill - 1
-    t = ill
-    s = 1
-    c = nextpow(2, groupsize()[1]) >> 1
+    @private m = ill - 1
```

Comment:
I suspect this form of `@private` is rather slow until KernelCompiler.jl lands. Using the older form is likely faster